conforming/README.md

```diff
@@ -0,0 +1,163 @@
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+
+-->
+
+# Tractus-X Conforming Agent (KA-CONF)
+
+KA-CONF is a module of the [Tractus-X Knowledge Agents Reference Implementations](../README.md).
+
+## About this Module
+
+The Conforming Agent is a component that may be used in place of Matchmaking and Binding Agents to prove the conformity of other components of the KA architecture.
+It implements (depending on its configuration) and validates the KA-MATCH and KA-BIND SPARQL profiles behind the Knowledge Agents standard API.
+It may also be used to provoque error behaviour in the KA dataflow in order to check robustness and conformity in the case of problems.
+It may also be used in unit test environments as a mock server.
+
+## Architecture
+
+The FOSS Conforming Agent is built using [Jersey](https://github.com/eclipse-ee4j/jersey) (for the REST endpoints) and [RDF4J](https://rdf4j.org/) (for SPARQL parsing). It exposes three endpoints for the three different SPARQL profiles which are to be certified/tested in the KA architecture:
+* The '/match' endpoint for the KA-MATCH profile
+* The '/bind' endpoint for the KA-BIND profile
+* The '/transfer' endpoint for the KA-TRANSFER profile
+
+The Conforming Agent does not perform any logic but just returns prerendered results depending on the input parameters.
+
+### Security
+
+The conforming agent will be configurable to support the following http/s based security mechanisms:
+- Basic Authentication
+- Oauth2
+
+## Deployment
+
+### Compile, Test & Package
+
+```console
+mvn package
+```
+
+This will generate 
+- a [plugin jar](target/original-conforming-agent-1.9.5-SNAPSHOT.jar) containing all necessary components to be dropped into a Jakarta-Compatible Web Server.
+- a [standalone jar](target/conforming-agent-1.9.5-SNAPSHOT.jar) including the Jakarta-Reference Implementation (Glassfish).
+
+### Run Locally
+
+The [standalone jar](target/conforming-agent-1.9.5-SNAPSHOT.jar) may be started as follows
+
+```console
+java -cp target/conforming-agent-1.9.5-SNAPSHOT.jar org.eclipse.tractusx.agents.conforming.Bootstrap"
+```
+
+### Containerizing
+
+You could either call
+
+```console
+mvn install -Pwith-docker-image
+```
+
+or invoke the following docker command after a successful package run
+
+```console
+docker build -t ghcr.io/eclipse-tractusx/knowledge-agents/conforming-agent:1.9.5-SNAPSHOT -f src/main/docker/Dockerfile .
```

Comment:
Do not use ghcr.io but Docker Hub. Please refer to [TRG 4.05 - Container registries](https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-05).

This comment addresses all other apprearances of ghcr.io as well.

Reply:
moved to dockerhub/tractusx